circle.yml

```diff
@@ -20,14 +21,14 @@ machine:
     GUSH_USE_FS: true
 dependencies:
   cache_directories:
-    - "~/docker"
+    - "$HOME/docker"
   override:
     - docker build -t gush .
 test:
   override:
-    - composer install
-    - wget https://phar.phpunit.de/phpunit.phar && chmod +x phpunit.phar
-    - php phpunit.phar --configuration phpunit.xml.dist --verbose
+    - composer install --prefer-dist --optimize-autoloader --no-interaction --no-scripts
+    - mkdir -p $CIRCLE_TEST_REPORTS/phpunit
+    - vendor/bin/phpunit --configuration phpunit.xml.dist --verbose --log-junit $CIRCLE_TEST_REPORTS/phpunit/junit.xml
     #- docker run -it -e GUSH_CONFIG=`cat ./.gush.yml | base64` gush -vvvv --help
```

Comment:
I've configured my fork to run Circle CI tests in order to debug the `.gush.yml` path via SSH, but the issue regarding "docker: Error parsing reference: ..." is now solved.